resources/home/dnanexus/run_workflows/utils/manage_dict.py

```
@@ -279,6 +280,7 @@
     project_name,
     sample=None,
     sample_prefix=None,
+    job_outputs_dict=None,
```

Ruff (0.8.2)

Missing type annotation for function argument `job_outputs_dict`

(ANN001)